src/sagemaker/local/image.py

```diff
@@ -526,16 +705,28 @@ def _create_docker_host(self, host, environment, optml_subdirs, command, volumes
         optml_volumes = self._build_optml_volumes(host, optml_subdirs)
         optml_volumes.extend(volumes)
 
+        container_name_prefix = "".join(
+            random.choice(string.ascii_lowercase + string.digits) for _ in range(10)
```

Comment:
`uuid` is possibility?

Reply:
Haven't changed it for consistency with other places where random job names, etc. are generated.